docs/mkdocs.yml

```diff
@@ -601,6 +601,7 @@ nav:
       - 'Sync SharePoint document library documents with Azure storage container': 'sample-scripts/spo/sync-splib-into-az-storage-container.md'
     - Microsoft Teams:
       - 'Create a Microsoft Team with channels from a Microsoft 365 Group': 'sample-scripts/teams/create-team-from-group.md'
+      - 'Create a Microsoft Teams team and bulk add members from CSV file': 'create-team-and-add-members-and-owners.md'
```

Comment:
Incorrect path to the script. Missing `sample-scripts/teams/` in front on the file path